### PR TITLE
Remove 'obsolete' that was added previously for python 3.6 package

### PR DESCRIPTION
**Pull request by Odilhao**
and remove files in /usr/bin

(cherry picked from commit 71a74dd1d1ef4b6417562323a946ce6ac6c88da4)

### CODE REVIEW

File: packages/python-sqlparse/python-sqlparse.spec [hunk 1]
```diff
@@ -70,6 +67,9 @@ set -ex
 * Tue Nov 09 2021 Odilon Sousa <osousa@redhat.com> - 0.4.2-1
```
Comment (evgeni) [inline packages/python-sqlparse/python-sqlparse.spec:70]: I think *this* one needs to be bumped to 0.4.2-2, as it's actually been updated in 3.16, the others need just be tagged

File: packages/python-sqlparse/python-sqlparse.spec [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 Name:           %{?scl_prefix}python-%{pypi_name}
 Version:        0.4.2
-Release:        1%{?dist}
+Release:        2%{?dist}
```
Comment (evgeni) [inline packages/python-sqlparse/python-sqlparse.spec:9]: please add a changelog entry for that one :)

Comment (Odilhao) [reply packages/python-sqlparse/python-sqlparse.spec:9]: Done, my bad

**Review by evgeni — APPROVED** [review]
ACK

sqlparse will need a build, everything else needs to be tagged from 3.15

if you tag first, and then merge this PR, jenkins will do the right thing and only build sqlparse :)

**Comment by Odilhao** [comment]
Done with all untags, merging this now

**Pull request merged**